### PR TITLE
YLP-443 DBL settings labels

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

**Comment by franck-fourel** [comment]
still need to validate https://github.com/mdblp/translations/pull/7 before merging this one

File: packages/viz/src/components/settings/common/Table.js [hunk 1]
```diff
@@ -61,7 +69,13 @@ class Table extends React.PureComponent {
 
   renderRow(normalizedColumns, rowKey, rowData) {
     const cells = _.map(normalizedColumns,
-      (column) => <td key={column.key}>{column.cell(rowData, column.key)}</td>
+      (column) => {
+        console.log(column);
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/common/Table.js:73]:
```suggestion
```

File: packages/viz/src/components/settings/common/Table.js [hunk 2]
```diff
@@ -61,7 +69,13 @@ class Table extends React.PureComponent {
...
+        const classname = (column.className)? `styles.secondaryLabelWithMain` : "styles.secondaryLabelWithMain";
+
+        return <td key={column.key} className={classname}>{column.cell(rowData, column.key)}</td>
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/common/Table.js:76]:
```suggestion
        return <td key={column.key} className={classname}>{column.cell(rowData, column.key)}</td>;
```

File: packages/viz/src/components/settings/common/Table.js [hunk 3]
```diff
@@ -49,6 +49,14 @@ class Table extends React.PureComponent {
             </th>
           );
         }
+        if (column.className){
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/common/Table.js:52]:
```suggestion
        if (column.className) {
```

File: packages/viz/src/components/settings/Diabeloop.js [hunk 4]
```diff
@@ -50,23 +50,29 @@ function renderDiabeloopParameters(parametersByLevel) {
...
+      className: 'secondaryLabelWithMainBold',
     },
   ];
 
   // eslint-disable-next-line lodash/prefer-lodash-method
   parametersByLevel.forEach((parameters, level) => {
     const title = {
       label: {
-        main: `${t('level')} ${level}`,
+        main: `${t('Advanced')}`,
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/Diabeloop.js:69]:
```suggestion
        main: level === 1 ? '' : `${t('Advanced')}`,
```

File: packages/viz/src/components/settings/Diabeloop.js [hunk 5]
```diff
@@ -50,23 +50,29 @@ function renderDiabeloopParameters(parametersByLevel) {
...
       },
       className: styles.bdlgSettingsHeader,
     };
+    if (level == "1" ) {
+      title.label.main = "";
+    }
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/Diabeloop.js:75]:
```suggestion
```

File: packages/viz/src/components/settings/common/Table.js [hunk 6]
```diff
@@ -61,7 +69,13 @@ class Table extends React.PureComponent {
 
   renderRow(normalizedColumns, rowKey, rowData) {
     const cells = _.map(normalizedColumns,
-      (column) => <td key={column.key}>{column.cell(rowData, column.key)}</td>
+      (column) => {
+        console.log(column);
+        const classname = (column.className)? `styles.secondaryLabelWithMain` : "styles.secondaryLabelWithMain";
```
Comment (yvon-dblg) [inline packages/viz/src/components/settings/common/Table.js:74]:
```suggestion
        const classname = (column.className) ? `styles.secondaryLabelWithMain` : "styles.secondaryLabelWithMain";
```

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Lint !
No it's not chocolate !

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**